src/RadForce/test_radiation_force.cpp

```diff
@@ -183,6 +215,15 @@ AMRSimulation<TubeProblem>::setCustomBoundaryConditions(const amrex::IntVect &iv
 	amrex::Real rho = NAN;
 	amrex::Real vel = NAN;
 
+	// CCH: calculate radEnergyFractions
```

Comment:
This is not strictly necessary, since future users can always find that you wrote this line of code with `git blame`.

Reply:
Resolved this and all similar comments.